app/helpers/application_helper.rb

```diff
@@ -3,4 +3,9 @@ def options_for_video_reviews(selected = nil)
     options = [5,4,3,2,1].map { |number| [pluralize(number, "Star"), number] }
     options_for_select(options, selected)
   end
+
+  def make_video_id(category, video)
+    category_name = category.name.downcase.sub(" ", "_")
+    "#{category_name}_#{video.title.underscore}"
+  end
```

Comment:
Where do you use this method? I could not find the use and it also looks strange to me. Is that the ID for JS ? You should check Rails Helpers like `dom_id`